lib/ResourceNamespace.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+function ResourceNamespace(stripe, resources) {
+  for (const name in resources) {
+    const camelCaseName = name[0].toLowerCase() + name.substring(1);
+    const resource = new resources[name](stripe);
+
+    this[camelCaseName] = resource;
+  }
+}
+
+module.exports = function(namespace, resources) {
+  return function(stripe) {
```

Comment:
you'll want to get rid of all of the stripe references :)

Reply:
oh yeah, 👍